src/include/uapi/ipc/dai-intel.h

```diff
@@ -181,8 +189,11 @@ struct sof_ipc_dai_dmic_params {
 
 	uint32_t num_pdm_active; /**< Number of active pdm controllers */
 
+	uint32_t wake_up_time;      /**< Time from clock start to data (us) */
+	uint32_t min_clock_on_time; /**< Min. time that clk is kept on (us) */
+
 	/* reserved for future use */
-	uint32_t reserved[8];
+	uint32_t reserved[6];
```

Comment:
@singalsu sorry, I didn realize we had some reserved here. So they can go after min_clock_on_time and before reserved.  The intention is that the struct upto the change offset will look just like the old one.